cpp/src/lib/osd_access.cc

```diff
@@ -117,16 +117,45 @@ int OsdAccess::read_osds_slices(
     osd_t osd = item.first;
     auto &osd_slices = item.second;
     // TODO this could be done in parallel
-    rc = _read_osd_slices(osd, osd_slices);
+    rc = _read_osd_slices_asd_direct_path(osd, osd_slices);
```

Comment:
this change means that when using the rora proxy client we'll currently always connect directly to the asd using the ocaml protocol and never via the gobjfs/libxio based protocol